docs/user-guide/layer-settings.md

```diff
@@ -55,7 +55,7 @@ Through the second section of the layer settings panel it is possible to change
 
 In particular, the user is allowed to:
 
-* Set the image format: choosing between `png`, `png8`, `jpeg`, `vnd.jpeg-png` and `gif` 
+* Set the image format: choosing between `png`, `png8`, `jpeg`, `vnd.jpeg-png`, `vnd.jpeg-png8` and `gif` 
```

Comment:
The list of available format is the sabe of the related catalog source. Also in this case a button allows to fetch the updated list of supported format from the server.

![image](https://user-images.githubusercontent.com/1280027/119498526-ff72eb00-bd65-11eb-9778-00c4c5da898f.png)